### ISSUE TITLE
[feature request] Ore dictionary support

### ISSUE THREAD

**Issue by Siphalor** [Martacus/Solar#14]
It would be nice if the recipes and the altar repair system used the ore dictionary, so you can use woods from other mods as well. ;-)

**Comment by Martacus** [comment]
Yes, right now it also doesn't take any other mods their silver nuggets and such. Indeed something I will need to add :P
